bm_scrapy/__main__.py

```diff
@@ -54,13 +54,15 @@ def setup_and_launch():
 
     
 def main():
-    # bm-crawl command
+    """bm-crawl command."""
+    """
     try:
```

Comment:
Is not a good practice to comment code, please fix and fix the docstring also

Reply:
Good advice, I'll change it.